### ISSUE TITLE
Remove or move Wildfly migration guide

### ISSUE THREAD

**Issue by shawkins** [keycloak/keycloak#33602]
### Description

The Wildfly migration guide is taking up prime real estate on https://www.keycloak.org/guides - given that we're nearing the 3 year mark since the last Wildfly release, it should no longer need to be prominent.

WDYT? @vmuzikar @stianst @ahus1

**Comment by vmuzikar** [comment]
I agree we should remove it from the main Guides page but at the same time I still do think we should have it archived somewhere (which is related to https://github.com/keycloak/keycloak/issues/33532#issuecomment-2393497328) as some folks are still probably using WF-based KC.

**Comment by shawkins** [comment]
Opened a pr to move the migration guide to the bottom of the list. I'll leave the removal for the versioning / archival issue.

**Comment by vmuzikar** [comment]
Actually, the [linked PR](https://github.com/keycloak/keycloak/pull/36011) did not really address this issue. @shawkins Should we reopen it?
